reflex-algos/src/main/python/mlcomp/README.md

```diff
@@ -1,34 +1,39 @@
 # README
 
-The `mlcomp` module is designed to process and run complex PySpark pipelines, which contain
-multiple PySpark components. These components can be uploaded by the user.
+The `mlcomp` module is designed to process and execute 'MCenter' complex pipelines,
```

Comment:
Don't use MCenter in mlpiper -- the pipeline should run independently of MCenter.